docs/install/conda.rst

```diff
@@ -17,9 +17,8 @@ Binary Package (For Users)
 Binary distributions of the latest release for mac and linux (64-bit) 
 using the conda package manager can be installed by running the command::
 
-    conda install -c cyclus -c pyne pyne=VERSION
+    conda install -c conda-forge pyne=0.5.3
```

Comment:
For future generality, how about:

    conda install -c conda-forge pyne=VERSION

where VERSION should be replaced with the version number to be installed (must be 0.5+).